### PR TITLE
Changed the Dune 2000 mod chooser description

### PR DESCRIPTION
**Pull request by Mailaender**
as we now have sandworms, carryalls and concrete this is not "Red Alert with sand" anymore. No need to degrade the mod in the description. The other ones focus on describing the game universe so adapt to that.

### CODE REVIEW

**Comment by obrakmann** [comment]
:+1:

File: mods/d2k/mod.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Metadata:
 	Title: Dune 2000
-	Description: Three great houses fight for the precious spice, melange.\nHe who controls the spice controls the universe!\n\nOur work in progress Dune 2000 recreation brings modern\ngameplay improvements into the Dune universe.\n\nFuture versions of OpenRA will incorporate more of the missing\nunique features from this classic game.
+	Description: Three great houses fight for the precious spice, melange.\nHe who controls the spice controls the universe!\n\nTry to establish a foothold on the desert planet Arrakis\nwith it's harsh environmental conditions.\n\nProtect your units from giant sandworms and other factions\nfighting over the precious resources.
```
Comment (Unit158) [inline mods/d2k/mod.yaml]: precious resources -> spice melange?

Comment (Unit158) [inline mods/d2k/mod.yaml]: also it's -> its

**Comment by Mailaender** [comment]
Updated.

**Comment by penev92** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/_compare/9b8a6555eac8f30d35914d52a69e691dc269dd09...7f8d9924a2da2c9dbe7f89df546c4cb505368001)

**Pull request merged**